ui/i18n/routing.ts

```diff
@@ -14,5 +14,13 @@ export const routing = defineRouting({
 
 // Lightweight wrappers around Next.js' navigation APIs
 // that will consider the routing configuration
-export const { Link, redirect, usePathname, useRouter } =
-  createSharedPathnamesNavigation(routing);
+let nav = createNavigation(routing);
+
+export const { Link, redirect, usePathname, useRouter } = {
+    Link: nav.Link,
+    redirect: (path: string) => {
+        nav.redirect({ href: path, locale: "en" });
+    },
+    usePathname: nav.usePathname,
+    useRouter: nav.useRouter,
+};
```

Comment:
The `redirect` function requires the locale on each use now. This is a cheat to use the default since we're only supporting `en` at the moment.